schema/url/standalone-map/plugin-standalone-map-markers.raml

```diff
@@ -34,13 +32,64 @@ types:
       mapElements: StandaloneMapElementInfo[]
 
   # TODO remove once quantile endpoint is merged.
+  NumericBinRange:
+    type: BinRange
+    additionalProperties: false
+    properties:
+      binLabel: string
+      binStart?: number # Dates can wired as millis since epoch. TODO is there a better raml-native way to model this? Trying not to overthink since this will be provided by quantile plugin.
+      binEnd?: number
+
+  DateBinRange:
+    type: BinRange
+    additionalProperties: false
+    properties:
+      binLabel: string
+      binStart?: string # Dates can wired as millis since epoch. TODO is there a better raml-native way to model this? Trying not to overthink since this will be provided by quantile plugin.
+      binEnd?: string
+
   BinRange:
     additionalProperties: false
     properties:
-      binStart?: string # number | date
-      binEnd?: string # number | date
       binLabel: string
 
+  OverlayType:
```

Comment:
i think my only question/ comment so far is about validation.. it seems like this puts more pressure on us to make sure people cant do things like pass us a date variable spec as overlay and then claim the overlay type is numeric or something?